app/services/real_performance_tracker.py

```
@@ -0,0 +1,369 @@
+
+                self.logger.info(
+                    "✅ Calculated REAL performance metrics",
+                    strategy_id=strategy_id,
+                    total_trades=len(trades),
+                    win_rate=metrics.get('win_rate'),
+                    total_pnl=metrics.get('total_pnl')
+                )
+
+                return metrics
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)